client/src/modules/plan/planner/planner.html

```diff
@@ -0,0 +1,17 @@
+<div class="planner-container flex">
+  <div class="flex flex-center">

```

Comment:
this the planner inner nav bar? give it a class like: planner-top-nav so we can style it